src/rules/noUnusedVariableRule.ts

```diff
@@ -32,12 +29,14 @@ class NoUnusedVariablesWalker extends Lint.RuleWalker {
     private skipBindingElement: boolean;
     private skipParameterDeclaration: boolean;
     private skipVariableDeclaration: boolean;
+
```

Comment:
please remove this extra newline, and move `private languageService` to the top (alphabetization)